protos/guard/guard.proto

```diff
@@ -243,6 +258,16 @@ message CheckFileStoreMetaRequest {
   bytes signature = 5; //requester signed
 }
 
+message FileInfoRequest {
+  string renter_pid = 1;
+  string requester_pid = 2;
+  google.protobuf.Timestamp request_time = 3 [
+    (gogoproto.nullable) = false,
+    (gogoproto.stdtime) = true
+  ];
+  bytes signature = 4;
```

Comment:
Need to support pagination if renter has a lot of files. See `ListHostContractsRequest`

Reply:
pagination was added in the parameters.